optaplanner-examples/pom.xml

```diff
@@ -49,6 +49,9 @@
         <configuration>
           <ignoredUsedUndeclaredDependencies>
             <dependency>org.optaplanner:optaplanner-core-impl:jar</dependency>
+            <!-- optaplanner-persistence-jackson brings dependencies we do not want to enumerate. -->
+            <dependency>com.fasterxml.jackson.core:*</dependency>
```

Comment:
It's not about what `optaplanner-persistence-jackson` brings; examples now use the Jackson annotations and we should declare these as our direct dependencies.

Reply:
The annotations, yes, maybe. But there are more JARs that than that this fails on.

Reply:
The other one is 'jackson-databing'; I believe we can get rid of it if we use the `OptaPlannerJacksonModule`.